marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/entity-services/test-data/content/json-to-xml.xml

```diff
@@ -0,0 +1,15 @@
+<envelope>
+  <instance>
+    <!-- this example has more than one element directly under instance to simulate JSON to XML ingest -->
+    <personId>222</personId>
+    <theNickname>Nicky</theNickname>
+    <theName>
+      <middleName>Middle</middleName>
+      <lastName>Last</lastName>
+      <firstName>
+        <theValue>First</theValue>
+        <thePrefix>SomePrefix</thePrefix>
+      </firstName>
+    </theName>
```

Comment:
Can we please add an array example to the test.

Reply:
@bsrikan Are you talking about the scenario for https://project.marklogic.com/jira/browse/DHFPROD-3731? I believe that is a separate issue.

Reply:
No, just to get more coverage just to make sure extractInstance works as expected for array of elements